docs/templates/applications.md

```diff
@@ -459,6 +461,62 @@ These weights are released under [the Apache License](https://github.com/tensorf
 
 -----
 
+## InceptionResNetV2
+
+
+```python
+keras.applications.inception_resnet_v2.InceptionResNetV2(include_top=True, weights='imagenet', input_tensor=None, input_shape=None, pooling=None, classes=1000)
+```
+
+Inception-ResNet V2 model, with weights pre-trained on ImageNet.
+
+This model is available for both the Theano and TensorFlow backend (but not CNTK), and can be built both
+with `channels_first` data format (channels, height, width) or `channels_last` data format (height, width, channels).
```

Comment:
Same

Reply:
Just to make sure, in current version (i.e. master branch) of this file, both "channels_first" (plain text) and \`channels_first\` (code keyword) are used. The code keyword format \`channels_first\` is used in the arguments section. Is it better to follow current usage, or unify them into one of "channels_first", \`channels_first\` or \`'channels_first'\`?

Reply:
Let's go with `'channels_first'`, I guess, for maximum explicitness